go/libraries/doltcore/sqle/sqldelete_test.go

```diff
@@ -189,14 +189,6 @@ func TestExecuteDeleteSystemTables(t *testing.T) {
 }
 
 var systemTableDeleteTests = []DeleteTest{
-	//{
```

Comment:
These tests should confirm that you can delete  / insert / update from the docs table